Xamarin.Forms.Core/TemplateBinding.cs

```diff
@@ -3,20 +3,17 @@
 
 namespace Xamarin.Forms
 {
-	public class TemplateBinding : BindingBase
+	// Theoretically this class shouldn't be necessary given
+	// Binding.RelativeSource and TemplateBindingExtension 
+	// but it's here to avoid breaking changes.
+	public sealed class TemplateBinding : Binding
```

Comment:
@StephaneDelcroix yeah I don't really love it myself, but I love redundant code less. :) Alternatively, we could just leave `TemplateBinding` alone and mark it as `Obsolete`. There's really no need for it since you have the `TemplateBindingExtension` for XAML and in code you can use regular `Binding` with `RelativeSource`. But of course we don't know if it's been used in the field.

Reply:
I agree, let's [Obsolete]